installer/install.py

```diff
@@ -268,42 +266,62 @@ def debug_wifi():
 
 report_success()
 
-debug_wifi()
 
-# see if we already have a repacked base filesystem
-basefw_squashfs_path = f"{boot_path}/images/{basefw_squashfs}"
-if not exists_with_md5(basefw_squashfs_path, basefw_squashfs_md5):
-    # maybe we can at least skip downloading it?
-    basefw_update_path = f"{boot_path}/firmware/{basefw_update}"
-    if not exists_with_md5(basefw_update_path, basefw_update_md5):
-        report_progress("Downloading base firmware")
-        if not ask_permission(f"Base firmware does not exist on the SD card: {basefw_update}<br><br>Download it from Bambu Lab servers?"):
-            report_failure(f"Base firmware {basefw_update} does not exist on the SD card.  Place it in {boot_path}/firmware and try again.")
+def download_firmware(update_url, dest_path):
+    if os.system("ping -c 3 -W 1000 8.8.8.8") != 0: 
```

Comment:
Idea: maybe try pinging the actual hostname of the basefw update, so we test DNS and connectivity to the actual DL endpoint for what we are downloading. Not a blocker, just a suggestion.

Reply:
That's a good idea, but @bani6809 has a good point above too. If there's an issue with connecting to the CDN link, we can determine that by checking how many bytes have been downloaded. 

The goal of this is just to identify whether a user has a working connection but can't download from CDN links (which was Issue #184)